lib/helpers/check-attributes.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const calculateLocationDisplay = require('./calculate-location-display');
+const supportsAttribute = require('./supports-attribute');
+
+module.exports = function checkAttributes(node, supported) {
+  let sourceReference = calculateLocationDisplay(null, node.loc.start);
+
+  if (node.modifiers.length > 0) {
+    throw new Error(
+      `Passing element modifiers to the <${
+        node.tag
+      }> component at ${sourceReference} is not supported by ember-angle-bracket-invocation-polyfill.`
+    );
+  }
+
+  let unsupported = node.attributes
+    .map(({ name }) => name)
+    .filter(name => name.charAt(0) !== '@' && !supportsAttribute(name, supported));
+
+  if (unsupported.length > 0) {
+    console.warn(
```

Comment:
Using `console.warn` is not ideal, I would rather use `console-ui` here (e.g. to support `--silent`), but I found no obvious way to pass `this.ui` from the addon's index.js to the AST plugin. But maybe I'm missing something?

Reply:
Yea, passing `this.ui` in here would also be an issue because then we can't parallelize the build (since the config for the plugin wouldn't be "structured cloneable"). Let's go with the warning for now, as I think we think this will be some what less common right?